cmake/onnxruntime_unittests.cmake

```diff
@@ -395,7 +395,7 @@ list(APPEND onnx_test_libs ${FS_STDLIB} debug ${onnxruntime_EXTERNAL_LIBRARIES_D
 
 
 add_executable(onnx_test_runner ${onnx_test_runner_src_dir}/main.cc)
-target_link_libraries(onnx_test_runner PRIVATE onnx_test_runner_common ${GETOPT_LIB_WIDE} ${onnx_test_libs})
+target_link_libraries(onnx_test_runner PRIVATE onnx_test_runner_common ${GETOPT_LIB_WIDE} ${onnx_test_libs} ${trt_link_libs})
```

Comment:
this should be in the execution provider specific unit test cmake file.

Reply:
I've moved it to onnxruntime_unittests_internal.cmake